picard/album.py

```diff
@@ -626,6 +636,43 @@ def _finalize_loading(self, error):
 
         if not self._requests:
             self._finalize_loading_album()
+            # Vorschlags-Popup anzeigen (falls vorhanden)
+            try:
+                self._show_tag_suggestions()
+            except Exception as exc:  # noqa: broad-except
+                log.error("Fehler beim Anzeigen der Tag-Vorschläge: %s", exc)
+
+    def _show_tag_suggestions(self):
+        """Zeigt einen Dialog mit Tag-Vorschlägen und wendet sie ggf. an."""
+        from PyQt6 import QtWidgets
+        from picard.i18n import gettext as _
+        config = get_config()
+        if not config.setting['show_tag_suggestions']:
+            return
+        suggestions = getattr(self, 'tag_suggestions', None)
+        if not suggestions:
+            return
+
+        lines = [f"{tag}: {value}" for tag, value in suggestions.items()]
+        msg = ("<b>" + _("Tag-Vorschläge gefunden") + "</b><br/><br/>" +
+               _("Für dieses Album wurden folgende Tags vorgeschlagen:") +
+               "<br/>" + "<br/>".join(lines) + "<br/><br/>" +
+               _("Möchten Sie die vorgeschlagenen Werte übernehmen?"))
```

Comment:
This is not how translations should be done, nor how variable data should be inserted. In other languages the order for fixed text may be different.